### PR TITLE
Backend

### LINKED ISSUE #6: Login Page

### LINKED ISSUE #10: Register Page

### LINKED ISSUE #12: Google Login

### LINKED ISSUE #13: Implement Audience login/register system

### LINKED ISSUE #14: Implement in-house APIs

### LINKED ISSUE #16: Api Security Authorization

### PR DESCRIPTION
**Pull request by reviseUC73**
merge code from backend to main

### CODE REVIEW

File: Frontend/src/App.jsx
Comment (natekrth) [inline Frontend/src/App.jsx:1]: Beware of Conflict when merge frontend branch to main

Comment (breezjirasak) [reply Frontend/src/App.jsx:1]: btw, is it a backend branch?

Comment (breezjirasak) [reply Frontend/src/App.jsx:1]: sry from misunderstand you mean on the folder frontend right?

**Comment by natekrth** [comment]
REMINDER @reviseUC73  @natekrth  @breezjirasak 
**BEWARE OF CONFLICT IN FRONTEND FILE WHEN WE MERGE FRONTEND BRANCH TO MAIN** 
This pull request also have some code change in Frontend file.

**Comment by breezjirasak** [comment]
Same as @natekrth beware when you include frontend into it.

**Review by natekrth — APPROVED** [review]
Good to go

File: Frontend/src/App.jsx
Comment (natekrth) [reply Frontend/src/App.jsx:1]: Yes, Breeze

**Review by breezjirasak — APPROVED** [review]
merge it !!!!!!!!!

**Review by Sankrub — APPROVED** [review]
good code easy to understand with comment

**Comment by reviseUC73** [comment]
Okay, I'm merging branches now.

**Pull request merged**